src/goto-programs/rw_set.cpp

```diff
@@ -30,6 +30,18 @@ void rw_sett::compute(const exprt &expr)
       assert(code.operands().size() == 1);
       read_rec(code.op0());
     }
+    else if (statement == "function_call")
+    {
+      assert(code.operands().size());
```

Comment:
We shouldn't restrict ourselves to only check verifier_assert, here's how cbmc handles it: https://github.com/diffblue/cbmc/blob/9dbbada7fa638b49c80242fe22f563933ca57d70/src/goto-instrument/rw_set.cpp#L57